app/analysis/index/controller.js

```diff
@@ -368,117 +345,113 @@ const AnalysisController = Controller.extend(AnalysisQueryParams.Mixin, {
       : [];
 
     return { projectAssignees, taskAssignees, customerAssignees };
-  }),
+  }
 
-  loadNext: task(function* () {
-    this.set("_shouldLoadMore", true);
+  @dropTask
+  *loadNext() {
+    this._shouldLoadMore = true;
 
     while (this._shouldLoadMore && this._canLoadMore) {
       yield this.data.perform();
 
       yield rAF();
     }
-  }).drop(),
-
-  download: task({
-    url: null,
-    params: {},
-
-    *perform(notify, allQueryParams, jwt, { url, params }) {
-      try {
-        this.setProperties({ url, params });
-
-        const queryString = toQueryString(
-          underscoreQueryParams(
-            cleanParams({
-              ...params,
-              ...serializeParachuteQueryParams(
-                allQueryParams,
-                AnalysisQueryParams
-              ),
-            })
-          )
-        );
+  }
+
+  @task
+  *download(notify, allQueryParams, jwt, { url = null, params = {} }) {
+    try {
+      this.url = url;
+      this.params = params;
+
+      const queryString = toQueryString(
+        underscoreQueryParams(
+          cleanParams({
+            ...params,
+            ...serializeParachuteQueryParams(
+              allQueryParams,
+              AnalysisQueryParams
+            ),
+          })
+        )
+      );
 
-        const res = yield fetch(`${url}?${queryString}`, {
-          headers: {
-            Authorization: `Bearer ${jwt}`,
-          },
-        });
-
-        /* istanbul ignore next */
-        if (!res.ok) {
-          throw new Error(res.statusText);
-        }
-
-        const file = yield res.blob();
-
-        // filename      match filename, followed by
-        // [^;=\n]*      anything but a ;, a = or a newline
-        // =
-        // (             first capturing group
-        //     (['"])    either single or double quote, put it in capturing group 2
-        //     .*?       anything up until the first...
-        //     \2        matching quote (single if we found single, double if we find double)
-        // |
-        //     [^;\n]*   anything but a ; or a newline
-        // )
-        const filename =
-          res.headers.map["content-disposition"]
-            .match(/filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/g)[0]
-            .replace("filename=", "") || "Unknown file";
-
-        // ignore since we can't really test this..
-        /* istanbul ignore next */
-        if (!Ember.testing) {
-          download(file, filename, file.type);
-        }
-
-        notify.success("File was downloaded");
-      } catch (e) {
-        /* istanbul ignore next */
```

Comment:
Are we actually not using codecoverage anymore in timed?

Reply:
It's currently deactivated. Since when: don't know. But we should activate it again soon. 

I deleted already some of those comments, since they annoyed me, but this was stupid I guess, since they are markers for codecov...